gdnative-derive/src/lib.rs

```diff
@@ -241,9 +241,93 @@ pub fn profiled(meta: TokenStream, input: TokenStream) -> TokenStream {
+/// The string returned from this method is displayed as a warning in the Scene Dock if the script that overrides it is a tool script.
+/// See [Godot docs](https://docs.godotengine.org/en/stable/classes/class_node.html#class-node-method-get-configuration-warning) for more information.
+///
+/// ```ignore
+/// fn _process(&mut self, owner:&T, f64);
+/// ```
+/// Called during processing step of the MainLoop.
+/// See [Godot docs](https://docs.godotengine.org/en/stable/classes/class_node.html#class-node-method-process) for more information.
+///
+/// ```ignore
+/// fn _physics_process(&self, owner: &Node, delta: f32);
+/// ```
+/// Called during physics update, with a fixed timestamp.
+/// See [Godot docs](https://docs.godotengine.org/en/stable/classes/class_node.html#class-node-method-physics-process) for more information.
+///
+/// ```ignore
+/// fn _input(&self, owner: &Node, event: InputEvent);
+/// ```
+/// Called when there is an input event.
+/// See [Godot docs](https://docs.godotengine.org/en/stable/classes/class_node.html#class-node-method-input) for more information.
+///
+/// ```ignore
+/// fn _unhandled_input(&self, owner: &Node, event: InputEvent);
+/// ```
+/// Called when an InputEvent hasn't been consumed by _input or any GUI.
+/// See [Godot docs](https://docs.godotengine.org/en/stable/classes/class_node.html#class-node-method-unhandled-input) for more information.
+///
+/// ```ignore
+/// fn _unhandled_key_input (&self, owner: &Node, event: InputKeyEvent);
+/// ```
+/// Called when an InputEventKey hasn't been consumed by _input or any GUI.
+/// See [Godot docs](https://docs.godotengine.org/en/stable/classes/class_node.html#class-node-method-unhandled-key-input) for more information.
+///
+/// #### `Control` Virtual Functions
+///
+/// This is a list of common Godot Virtual functions that are automatically called via [notifications](https://docs.godotengine.org/en/stable/classes/class_object.html#class-object-method-notification).
+///
+/// ```
+/// fn _clips_input(&self, owner: &Control) -> bool;
+/// ```
+/// Returns whether _gui_input should not be called for children controls outside this control's rectangle.
+/// See [Godot docs](https://docs.godotengine.org/en/stable/classes/class_control.html#class-control-method-clips-input) for more information.
+///
+/// ```
+/// fn _get_minimum_size(&self, owner: &Control) -> Vector2;
+/// ```
+/// Returns whether _gui_input should not be called for children controls outside this control's rectangle.
+/// See [Godot docs](https://docs.godotengine.org/en/stable/classes/class_control.html#class-control-method-get-minimum-size) for more information.
+///
+/// ```
+/// fn _gui_input(&self, owner: &Control, event: InputEvent);
+/// ```
+/// Returns whether _gui_input should not be called for children controls outside this control's rectangle.
+/// See [Godot docs](https://docs.godotengine.org/en/stable/classes/class_control.html#class-control-method-gui-input) for more information.
```

Comment:
Description is identical 3x

Reply:
Well that's embarrassing. That's what I get for focusing on the links instead of the actual content :rofl:
Fixed this up as well.